modules/test/playwright/tests/account-admin-web/accountPermissions.spec.ts

```diff
@@ -311,3 +315,118 @@ test.describe('Test for Organization Account visibility depending on Permissions
+	test('LPD-30009 Account admin can unassign organization to account', async ({
+		accountManagementWidgetPage,
+		accountOrganizationsPage,
+		accountsPage,
+		apiHelpers,
+		page,
+	}) => {
+		page.on('dialog', (dialog) => dialog.accept());
+
+		const userName = 'demo.unprivileged';
+		const accountAdministrator = 'Account Administrator';
+
+		await page.goto('/search');
+
+		await accountManagementWidgetPage.addAccountManagementWidget();
+
+		try {
+			const account = await apiHelpers.headlessAdminUser.postAccount({
+				name: 'Account' + getRandomInt(),
+				type: 'business',
+			});
+
+			apiHelpers.data.push({id: account.id, type: 'account'});
+
+			await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+				account.id,
+				[`${userName}@liferay.com`]
+			);
+
+			const rolesResponse =
+				await apiHelpers.headlessAdminUser.getAccountRoles(account.id);
+
+			const accountAdminRole = rolesResponse?.items?.filter(
+				(role) => role.name === accountAdministrator
+			);
+
+			const userAccount =
+				await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+					`${userName}@liferay.com`
+				);
+
+			await apiHelpers.headlessAdminUser.assignUserToAccountRole(
+				account.id,
+				accountAdminRole[0].id,
+				userAccount.id
+			);
+
+			const organization =
+				await apiHelpers.headlessAdminUser.postOrganization();
+
+			await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+				account.id,
+				organization.id
+			);
+
+			await performLogout(page);
+			await performLogin(page, `${userName}`);
+
+			await page.goto('/search');
+
+			await page.getByText(account.name).click();
+			await accountsPage.organizationsTab.click();
+
+			await accountManagementWidgetPage.waitForSearchInput();
+
+			await expect(accountsPage.newButton).toHaveCount(0);
+
+			await performLogout(page);
+			await performLogin(page, 'test');
+
+			const companyId = await page.evaluate(() => {
+				return Liferay.ThemeDisplay.getCompanyId();
+			});
+
+			await apiHelpers.headlessAdminUser.putRole(accountAdministrator, {
+				name: accountAdministrator,
+				rolePermissions: [
+					{
+						actionIds: ['MANAGE_ORGANIZATIONS'],
+						primaryKey: companyId,
+						resourceName: 'com.liferay.account.model.AccountEntry',
+						scope: 1,
+					},
+				],
+				roleType: 'organization',
+			});
+
+			await performLogout(page);
+			await performLogin(page, `${userName}`);
+
+			await page.goto('/search');
+
+			await page.getByText(account.name).click();
+			await accountsPage.organizationsTab.click();
+
+			await accountOrganizationsPage.selectAllItemsCheckbox.click();
+			await accountOrganizationsPage.removeButton.click();
+
+			await accountManagementWidgetPage.waitForSearchInput();
+
+			await page.goto('/search');
+			await expect(page.getByText(organization.name)).toHaveCount(0);
+		}
+		finally {
+			await performLogout(page);
+			await performLogin(page, 'test');
```

Comment:
also if we move the creation of the page using api we still need these two lines because otherwise the "tear down" is done with the wrong user and the deletions fail